examples/inverse/evoked_ers_source_power.py

```diff
@@ -40,28 +41,28 @@
 # crop to 5 minutes to save memory
 raw = mne.io.read_raw_fif(raw_fname).crop(0, 300)
 
-# We are interested in the beta band (12-30 Hz)
+# we are interested in the beta band (12-30 Hz)
```

Comment:
I just changed things to be consistent within this tutorial with proper capitalization for rendered sections and lower case for inline comments. I hear you but the style inconsistencies are not good and make it hard to contribute new prose when the style is jumping around and you have no idea which one to use.

Reply:
@alexrockhill it's a trade off. The bottleneck is often the review time so anything you can do to reduce it will help. Basically step in my shoes. If I have 15 mins to review your code any minute counts and things that are not critical should be avoided.